tests/functional/test_related_sc_data/bundle_configs/A_X_to_A_Y/cases/case_4.yaml

```diff
@@ -1,5 +1,5 @@
-description: Attempt to save HC map with A.X and A.Y on first host and A.Y on second host- should fail
-positive: False
+description: Attempt to save HC map with A.X and A.Y on first host and A.Y on second host
```

Comment:
Information about test expectations is useful. Especially in the test description

Reply:
Fixed in split test cases